src/Containers/Microsoft.NET.Build.Containers/Resources/Strings.resx

```diff
@@ -158,7 +158,7 @@
     <comment>{StrBegin="CONTAINER3002: "}</comment>
   </data>
   <data name="DockerProcessCreationFailed" xml:space="preserve">
-    <value>CONTAINER3001: Failed creating docker process.</value>
+    <value>CONTAINER3001: Failed creating {0} process.</value>
     <comment>{StrBegin="CONTAINER3001: "}</comment>
```

Comment:
Can you also add a comment describing what `{0}` is here for the localization team? Something like "{0} is the name of the command we failed to run, usually 'docker' or 'podman'".